File: Pythonleetcode/Leetcode125.py
class Solution(object):
    def isPalindrome(self, s):
        k=''
        for i in range(0,len(s)):
            if s[i].isdigit() or s[i].isalpha():
                k=k+s[i]
        k=k.upper()
        for i in range(0,len(k)):
            if k[i]!=k[len(k)-1-i]:
                return False
        return True



print(Solution().isPalindrome("A man, a plan, a canal: Panama"))

File: CppPractice/CppLeetcode/Leetcode125.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string newString="";
        for(char c:s)
        {
            if(c>='A'&&c<='Z'){c=c-'A'+'a';newString+=c;}
            else if((c>='a'&&c<='z')||(c>='0'&&c<='9')){newString+=c;}
        }
        for(int i=0;i<newString.size()/2;i++)
        {
            if(newString[i]!=newString[newString.size()-1-i]){return false;}

        }
        return true;
    }
};